src/index.ts

```diff
@@ -78,6 +84,7 @@ const createLoginHandler = ({ msalClient, scopes, authReplyRoute }: AuthInput):
           redirectUri: createReplyUrl(req, authReplyRoute),
           codeChallenge: pkceCodes.challenge,
           codeChallengeMethod: pkceCodes.challengeMethod,
+          ...authorizationUrlRequestOverride(req),
```

Comment:
Is that notation going to throw if consumers don't pass the option?

Reply:
Last time I tried this from memory it worked, but it's a good callout, need to check

Reply:
My IDE showed this as a linting error that `authorizationUrlRequestOverride` may be undefined.

I added a [couple of tweaks](https://github.com/MakerXStudio/express-msal/pull/7/commits/94f3afe9ef84fb3f1c5ea8c93be455bab8112cb7) to:
- optionally call the callback
- allow overriding everything except the `codeChallenge`, `codeChallengeMethod` and `redirectUrl`

Feel free to roll back or change if this is not what you intended!